tests/test_rules.py

```diff
@@ -806,6 +804,25 @@ def test_title_in_first_line(self):
         self.assertEqual(faulty_rules, [], Fore.RED +
                          "There are rules without the 'title' attribute in their first line.")
 
+    def test_duplicate_titles(self):
+        # This test ensure that every rule has a unique title
+        faulty_rules = []
+        titles_dict = {}
+        for file in self.yield_next_rule_file_path(self.path_to_rules):
+            title = self.get_rule_part(file_path=file, part_name="title").lower().rstrip()
+            duplicate = False
+            for rule, title_ in titles_dict.items():
+                if title == title_:
+                    print(Fore.RED + "Rule {} has an already used title in {}.".format(file, rule))
+                    duplicate = True
+                    faulty_rules.append(file)
+                    continue
+            if not duplicate:
+                titles_dict[file] = title
+
+        self.assertEqual(faulty_rules, [], Fore.RED +
+                         "There are rules with already used 'title'. Please check: https://github.com/SigmaHQ/sigma/wiki/Rule-Creation-Guide#title")
```

Comment:
```suggestion
                         "There are rules that share the same 'title'. Please check: https://github.com/SigmaHQ/sigma/wiki/Rule-Creation-Guide#title")
```
Also: You give a link but we never explicitly said that title has to be unique (and the link target also does not say anything about that, maybe we should change that as well).

I think there can be an argument to enforce this, but we didn't make that decision yet, did we? In my opinion the benefits outweigh the drawbacks.

Reply:
We have to add a information for this repot , somethink like : 

This depot use the Sigma specification (ref) with some enforcement
- `id` must be unique
- `title` must be unique

What do you think ?

Reply:
Yes, it's never stated but this test implies it. Also, this issue #3727 raised this because non-unique titles could create rare issues. (As you can see by the PR linked, we fixed all of them).

Now it's just a matter of adding that to the specs.